src/main/java/edu/usf/sas/pal/muser/model/MusicAnalysisModel.java

```diff
@@ -30,16 +30,201 @@ public MusicAnalysisModel() {
     public static class AlbumArtistData {
 
         public AlbumArtistData() {
+
```

Comment:
Let's try to keep the constructor declaration consistent to the previous implementation. Please remove the extra line